storage/rocksdb/ha_rocksdb.cc

```diff
@@ -19399,13 +19454,38 @@ static bool parse_fault_injection_params(
   return false;
 }
 
-bool ha_rocksdb::get_se_private_data(dd::Table *, bool reset) {
-  // TODO: Remove the assert once we fully migrate to using RocksDB DD API.
-  assert(false);
+bool ha_rocksdb::get_se_private_data(dd::Table *dd_table, bool reset) {
```

Comment:
will this func be called during restart(instead of boostrap)?

Reply:
> will this func be called during restart(instead of boostrap)? 

Only once for `dd_properties` (as opposed to all the tables during bootstrap)

> also will it call for non-dd tables?

No